csfieldguide/templates/interactives/scene-editor.html

```diff
@@ -0,0 +1,99 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+  <div class="container">
+    <div class="row mt-5">
+      <div id="scene" class="col-12"></div>
+    </div>
+
+    <div class="row my-3">
+      <div id="object-container" class="col-12 col-sm-4 text-center d-none">
+        <h5 class="heading-line heading-line-text">{% trans "New object:" %}</h5>
+        <form>
+          <div class="form-group">
+            <select id="object-selection" class="custom-select">
+              <option value="sphere">{% trans "Sphere" %}</option>
+              <option value="cube">{% trans "Cube" %}</option>
+              <option value="cone">{% trans "Cone" %}</option>
+            </select>
+          </div>
+          <div class="form-group">
+            <input type="text" maxlength="20" class="form-control" id="name-input" placeholder="{% trans 'Name (optional)' %}">
+          </div>
+          <div class="form-group">
+            <div class="input-group my-2">
+              <div class="input-group-prepend">
+                <label for="colour-input" id="colour-input-label" class="input-group-text">#</label>
+              </div>
+              <input type="text" maxlength="6" class="form-control" aria-describedby="random-colour-help" id="colour-input" placeholder="{% trans 'Colour (Hexadecimal)' %}">
+            </div>
+            <small id="random-colour-help" class="form-text text-muted">{% trans "Leave blank for a random colour" %}</small>
+          </div>
+          <button id="add-object" class="col-12 btn btn-primary" type="button">{% trans "Add object" %}</button>
```

Comment:
Would be nice if we could also remove objects. Do you think this is needed for MVP though?

Reply:
Not going to be an easy addition. I'll add it to #1154